components/meta-packages/X-incorporation/Makefile

```diff
@@ -36,38 +36,8 @@ clean:
 	$(RM) -r $(BUILD_DIR) $(PROTO_DIR)
 
 # Auto-generated dependencies
-REQUIRED_PACKAGES += compatibility/packages/SUNWxwinc
```

Comment:
Is it really auto-generated dependencies? I thought  there would be no REQUIRED_PACKAGES for this component.

Reply:
Yes, you can safely remove them all.
Otherwise looks good.

Reply:
OK I will vacuum with the next PR.